### PR TITLE
[RSHAM] add statistic for nature's swiftness mana save

### PR DESCRIPTION
**Pull request by Texleretour**
### Description

Added a new Statistic to track mana saved from restoration shaman's talent Nature's Swiftness
Data is 10x'ed during prepatch due to new hardcoded mana cost values

### CODE REVIEW

**Comment by Texleretour** [comment]
Resto shaman TWW support #6944 : Module in Statistics : Mana saved by Nature's swiftness

**Comment by Texleretour** [comment]
image

File: src/analysis/retail/shaman/restoration/modules/talents/NaturesSwiftness.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  manaSaved = 0;
+  castCount = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(TALENTS_SHAMAN.NATURES_SWIFTNESS_TALENT);
+
+    if (!this.active) {
+      return;
+    }
```
Comment (Krealle) [inline src/analysis/retail/shaman/restoration/modules/talents/NaturesSwiftness.tsx]: This can be removed, the module won't run if `this.active` is false.

**Review by Krealle — CHANGES_REQUESTED** [review]
Good first PR, just have a single comment.

File: src/analysis/retail/shaman/restoration/modules/talents/NaturesSwiftness.tsx (hunk 1 above)
Comment (Texleretour) [reply src/analysis/retail/shaman/restoration/modules/talents/NaturesSwiftness.tsx]: okay thanks ! done

**Review by Krealle — APPROVED** [review]

**Comment by Krealle** [comment]
LGTM

**Pull request merged**